extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/support/FastModel.java

```diff
@@ -46,6 +47,16 @@ public FastModel(CamelContext camelContext) {
         super(camelContext);
     }
 
+    @Override
+    public void addRouteDefinitions(InputStream is) throws Exception {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public void addRestDefinitions(InputStream is, boolean addToRoutes) throws Exception {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Maybe we can remove those two methods from Model's interface in RC2 and re-factorize them as helpers ?